setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -1196,6 +1196,17 @@ public function updateModulesSequence($keepGeneratedFiles = false)
         $this->createModulesConfig([]);
     }
 
+    /**
+     * Get the modules config as Magento sees it
+     *
+     * @return array
+     * @throws \LogicException
+     */
+    public function getModulesConfig()
```

Comment:
Could you please avoid the creation of new public methods.
I would recommend adding this implementation as a private method of `Magento\Setup\Console\Command\ ModuleConfigStatusCommand` class.

Reply:
The problem is that the `createModulesConfig` function is private, so I need to expose it somehow without copying it over completely.
According to the [semver checker result](https://results.solutionascode.com/results/magento/magento2/pull/22733/ea76f902-95f0-491c-82a7-5922f485cf57/10891/SemanticVersionChecker/report-magento2.html) it is ok to add a public function to this class.
What do you think?

Reply:
> The problem is that the createModulesConfig function is private, so I need to expose it somehow without copying it over completely.
You are right, I didn't notice this point.
So let's keep this method.
Thank you.